### ISSUE TITLE
Adding keybindings

### ISSUE THREAD

**Issue by jpspadeto2** [pemsley/coot#138]
Hi, now I'm using Coot 1 and it's a very powerful tool with a lot of improvements.

My question is, is there a way to add keybindings? I know there are some default keybindings, but for example, a keybinding to add water?

**Comment by pemsley** [comment]
> Hi, now I'm using Coot 1 and it's a very powerful tool with a lot of improvements.

You are invited to the next party.

> My question is, is there a way to add keybindings?

I should do a tutorial on this too...

```
import coot

def add_water_here():
    coot.place_typed_atom_at_pointer("Water")

coot.add_key_binding_gtk4_py(54, 0, add_water_here, "Add Water Here") # '6' key
```
The first argument is the key-code, the second is control, the third is the function and the 4th is a label.

This should have worked - but I just tried it and no atom is added. I will fix this bug shortly.

**Comment by jpspadeto2** [comment]
Nice, thank you. 

I’m changing between coot 0.9 and 1.0 yet due to water keybinding and preferences saving (i tried to save some parameters, but every time i open coot 1, restart to default settings). But the others functions are very nice and accordingly to pdb validate, very intuitive!

**Comment by pemsley** [comment]
This should be fixed now 8fbbf50aaa88e1ff05dd89dc4a581ef0d4b951bd.

**Comment by pemsley** [comment]
Thanks for keeping track
